packages/manager/modules/anthos/src/dashboard/host/remove/template.html

```diff
@@ -0,0 +1,19 @@
+<oui-modal
+    data-primary-action="$ctrl.removeHost()"
+    data-primary-label="{{ ::'anthos_dashboard_remove_host_submit' | translate }}"
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{ ::'anthos_dashboard_remove_host_cancel' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-heading="{{:: 'anthos_dashboard_remove_host_title' | translate}}"
+    data-type="warning"
```

Comment:
Add `loading` since there is an API call involved

Reply:
Updated.